xsd/MessaggiTypes_v1.1.xsd

```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I have no idea why https://www.fatturapa.gov.it and https://www.agenziaentrate.gov.it persevere to be unaligned.

Please add also `MessaggiFatturaTypes_v1.0.xsd` from the [main page](https://www.agenziaentrate.gov.it/wps/content/Nsilib/Nsi/Schede/Comunicazioni/Fatture+e+corrispettivi/Fatture+e+corrispettivi+ST/ST+invio+di+fatturazione+elettronica/?page=schedecomunicazioni).
Even though it appears to be only an older version, it still is the offical one from gov.it :confused: